index.html

```diff
@@ -122,19 +126,21 @@ <h2>MY RECENT<br>WORK</h2>
     <div class="post">
       
       <img class="img-post" src="./images/Snapshoot_Portfolio@2x.png" alt="Here goes and image">
-      <div class="post-title">Title</div>
-      <div class="post-desc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.  
+      <div class="textos">
+        <div class="post-title">Multi-Post Stories</div>
+        <div class="post-desc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.  
+        </div>
+        <div class="ellipse"></div>
+        <div class="languages">
+          <p>css</p><div class="updivisor"></div>
+          <p>html</p><div class="updivisor"></div>
+          <p>Bootstrap</p><div class="updivisor"></div>
+          <p>Ruby</p>
+        </div>
+        <button class="seebutton">See project</button>
       </div>
-      <div class="ellipse"></div>
-      <div class="languages">
-        <p>css</p><div class="updivisor"></div>
-        <p>html</p><div class="updivisor"></div>
-        <p>Bootstrap</p><div class="updivisor"></div>
-        <p>Ruby</p>
-      </div>
-      
     </div>
-    <button class="seebutton">See project</button>
+    
     
 
   </section>
```

Comment:
- [ ] Please make sure that  your contact button looks similar to the Figma template, currently one of your buttons has different styling. :+1: 

- [ ] Please ensure taht the email after the social media icon links shares the same row as the social media icon links. :smile: 

### Figma template
![Screenshot from 2022-07-12 23-08-29](https://user-images.githubusercontent.com/6615394/178613818-9068bf69-e58a-4885-8a3f-08b8bf7563f1.png)

### Your project
![Screenshot from 2022-07-12 23-08-11](https://user-images.githubusercontent.com/6615394/178613819-23ce7db8-b603-41ab-ac9a-6fa49279707c.png)